.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -160,6 +160,15 @@ jobs:
           TEST_DEVICE: CPU
           USE_SYSTEM_CACHE: False
 
+      - name: RoPE Test
```

Comment:
should we generalize this testing for any transformation? We can specify a list of transformations that should be applied for each model in the list:
<model_name>,<list of transformations>